docs/blog/_posts/2017-10-02-fourth-dotty-milestone-release.md

```diff
@@ -0,0 +1,141 @@
+manipulating ASTs (Abstract Syntax Trees), this is more robust and will make it
+easier to develop new features: we have already implemented auto-completion
+support (by reusing the APIs we had created for
+the [Dotty IDE](http://dotty.epfl.ch/docs/usage/ide-support.html)) and we have
+plans for displaying API documentation in the REPL.
+
+Note that the user interface of the REPL has not changed: like in the old REPL
+we use code adapted from the [Ammonite REPL](http://ammonite.io/#Ammonite-REPL)
+to provide syntax highlighting, multi-line editing, history, etc.
+
+### Scala 2.12 support [#2827](https://github.com/lampepfl/dotty/pull/2827)
+Since our first release, it has been possible to use Scala 2 libraries in a
+Dotty project as explained in the
+[dotty-example-project](https://github.com/smarter/dotty-example-project#getting-your-project-to-compile-with-dotty).
+Previously, we supported libraries compiled by Scala 2.11, but starting with this
+release we support Scala 2.12 instead. If your Dotty project has Scala 2
+dependencies this change should be transparent for you assuming all your
+dependencies have been published for 2.12.
+
+### Performance work
+Over the last few weeks, we started working on compilation speed with some good results:
+compiling [ScalaPB](https://github.com/dotty-staging/scalapb) is now 20% faster
+than with Dotty 0.3.0-RC2. You can follow along our progress on
+http://dotty-bench.epfl.ch/.
+
+
+## Trying out Dotty
+### Scastie
+[Scastie], the online Scala playground, supports Dotty.
+This is an easy way to try Dotty without installing anything.
+
+### sbt
+Using sbt 0.13.13 or newer, do:
+
+```
+sbt new lampepfl/dotty.g8
+```
+
+This will setup a new sbt project with Dotty as compiler. For more details on
+using Dotty with sbt, see the
+[example project](https://github.com/lampepfl/dotty-example-project).
+
+### IDE support
+It is very easy to start using the Dotty IDE in any Dotty project by following
+the [IDE guide](http://dotty.epfl.ch/docs/usage/ide-support.html).
+
+
+### Standalone installation
+Releases are available for download on the _Releases_
+section of the Dotty repository:
+[https://github.com/lampepfl/dotty/releases](https://github.com/lampepfl/dotty/releases)
+
+We also provide a [homebrew](https://brew.sh/) package that can be installed by running:
+
+```
+brew install lampepfl/brew/dotty
+```
+
+In case you have already installed Dotty via brew, you should instead update it:
+
+```
+brew upgrade dotty
+```
+
+## Let us know what you think!
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+## Contributing
+Thank you to all the contributors who made this release possible!
+
+According to `git shortlog -sn --no-merges 0.3.0-RC1..0.4.0-RC1` these are:
+
+```
+   TODO
```

Comment:
I presume the PR to scala-lang will include this? ;-)

Reply:
Waiting for a couple more PRs to get in before I tag the release. Will definitely be in the scala-lang PR =)